src/ts/forwarders/forwarder.ts

```diff
@@ -0,0 +1,227 @@
+import { ChainID } from "../..";
+import { ReplayProtectionAuthority } from "../replayProtection/replayProtectionAuthority";
+import { Wallet } from "ethers";
+import {
+  defaultAbiCoder,
+  BigNumberish,
+  arrayify,
+  keccak256,
+  solidityKeccak256,
+  getCreate2Address,
+} from "ethers/utils";
+import { Create2Options } from "ethers/utils/address";
+
+export interface EncodedTx {
```

Comment:
I dont think this name is very good. The interface should describe what it is, not how it's used. I would call this something like MinimalTransaction? Also not a very good name, but better than EncodedTx I think.

Reply:
but the transaction is encoded? e.g. it just has the address, calldata, and the gas that might be used?